projects/components/src/quick-search/quick-search.component.ts

```diff
@@ -628,4 +636,16 @@ export class QuickSearchComponent {
             searchTerm: removedFiltersSearch,
         };
     }
+
+    groupSectionTrackBy(_index: number, record: GroupedSearchSections): string {
+        return record.headerTitle;
+    }
+
+    sectionTrackBy(_index: number, record: SearchSection): string {
```

Comment:
Instead of `record: SearchSection` better rename to match the others `searchSection: SearchSection`